spec/debugger.spec.js

```diff
@@ -169,8 +169,7 @@ describe('equality-with-mountebank', () => {
 
     await fetch(mockedHttpBaseUrl + '/example?page=1&status=failed2-this-will-not-be-matched', {
       headers: {
-        'X-Something-Custom': 'abcdefg',
-        'user-agent': 'node'
```

Comment:
This just makes sure these tests don't fail on the `user-agent` being incorrect, which seems to happen sometimes, but isn't important for the test

Reply:
The tests were passing for me locally, and on CI.  I've updated this anyway.  As the assertion is about the headers being visible in the debugger I've opted for being specific about the headers on that request rather than leaving some for the HTTP client to add implicitly.